app/mixins/components/can-trigger-build.js

```diff
@@ -0,0 +1,18 @@
+import Mixin from '@ember/object/mixin';
+import { computed } from '@ember/object';
+import { equal, or, reads } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+
+export default Mixin.create({
+  features: service(),
+
+  canCreateRequest: reads('repo.permissions.create_request'),
+  proVersion: reads('features.proVersion'),
+  enterpriseVersion: reads('features.enterpriseVersion'),
+  proOrEnterprise: or('proVersion', 'enterpriseVersion'),
```

Comment:
hmmm, i'm not sure ... `version` for me would be a non-boolean value, wouldn't it? and it could have the value `org`, too, as opposed to `pro` (which should be `com`, but yeah) and `enterprise`.

imo the condition

```
this.canCreateRequest && (this.proOrEnterprise || !this.repoMigrated)
```

wouldn't read as well if we'd have

```
this.canCreateRequest && (this.version || !this.repoMigrated)
```

what do you think?